src/bootstrap/src/lib.rs

```diff
@@ -655,27 +655,37 @@ impl Build {
     }
 
     /// Gets the space-separated set of activated features for the standard
-    /// library.
+    /// library. This can be configured with the `std_features` key in config.toml.
     fn std_features(&self, target: TargetSelection) -> String {
-        let mut features = " panic-unwind".to_string();
```

Comment:
Before this PR the output of this function starts with a leading space. I don't know if this is actually required somewhere or not, but if it is, this PR will have to preserve that leading space.

Reply:
I think the space is required, but instead of adding it here, I have formatted it at the caller of `std_features` in `build_steps/compile.rs`

```rs
  let builder_std_features = format!(" {}", &builder.std_features(target));

features += &builder_std_features;
```